pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -0,0 +1,115 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, fetchpatch
+, fetchurl
+, pkg-config
+, cmake
+, python3
+, libpng
+, zlib
+, eigen
+, protobuf
+, howard-hinnant-date
+, nlohmann_json
+, boost
+, oneDNN
+, gtest
+}:
+
+let
+  abseil = fetchurl {
```

Comment:
Could you add a comment explaining this is kept in sync with an upstream CMake file, and patched in `postPatch`?

Just so it's a bit more clear we might need to check the version of this when updating the package in the future, since it's `url` attribute is directly referenced in the source code.

Reply:
If upstream CMake file changes then substituteInPlace will fail. I would move abseil back to postPatch for better readability.

Reply:
If upstream CMake file changes then substituteInPlace will fail. I would move abseil back to postPatch for better readability.

Reply:
I think that makes it more difficult to see that this is the case. You should add a comment explaining it instead.

Reply:
Done